tigerpath/majors_and_certificates/majors/GER_2018.json

```
@@ -0,0 +1,403 @@
+                  "course_list": [
+                    "GER 3*",
+                    "GER 4*",
+                    "GER 5*"
+                  ]
+                },
+                {
+                  "name": null,
+                  "max_counted": 2,
+                  "min_needed": null,
+                  "explanation": null,
+                  "course_list": [
+                    "GER 2*"
+                  ]
+                }
+              ]
+            },
+            {
+              "name": "Cognates",
+              "max_counted": 3,
+              "min_needed": null,
+              "explanation": "A maximum of three relevant cognate courses in History, European Cultural Studies, Religion, or Philosophy.",
+              "course_list": [
+                "HIS *",
+                "ECS *",
+                "PHI *",
+                "REL *"
+              ]
+            }
+          ]
+        },
+        {
+          "name": "Media & Aesthetics",
+          "max_counted": 1,
+          "min_needed": 8,
+          "explanation": "Designed for students who wish to focus on art, film, music, sound technology, and/or media theory broadly conceived. Students take a minimum of five courses in the German Department (usually no more than two at the 200 level), and a maximum of three relevant cognate courses in Art & Archaeology, Music, Philosophy, European Cultural Studies, and the Program in Visual Arts.",
+          "req_list": [
+            {
+              "name": "German",
+              "max_counted": null,
+              "min_needed": 5,
+              "explanation": "Students take a minimum of five courses in the German Department (usually no more than two at the 200 level).",
+              "req_list": [
+                {
+                  "name": null,
+                  "max_counted": null,
+                  "min_needed": null,
+                  "explanation": null,
+                  "course_list": [
+                    "GER 3*",
+                    "GER 4*",
+                    "GER 5*"
+                  ]
+                },
+                {
+                  "name": null,
+                  "max_counted": 2,
+                  "min_needed": null,
+                  "explanation": null,
+                  "course_list": [
+                    "GER 2*"
+                  ]
+                }
+              ]
+            },
+            {
+              "name": "Cognates",
+              "max_counted": 3,
+              "min_needed": null,
+              "explanation": "A maximum of three relevant cognate courses in Art & Archaeology, Music, Philosophy, European Cultural Studies, and the Program in Visual Arts.",
+              "course_list": [
+                "ART *",
+                "MUS *",
+                "PHI *",
+                "ECS *",
+                "VIS *"
+              ]
+            }
+          ]
+        },
+        {
+          "name": "German Linguistics",

```

This seems to be the official name according to the department.
```suggestion
          "name": "Germanic Linguistics",
```